modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ImportTaskResourceTest.java

```diff
@@ -410,6 +449,32 @@ public void testPostImportTask() throws Exception {
 			JSONCompareMode.LENIENT);
 	}
 
+	private void _executeDeleteImportTask(
+			ImportTaskResource importTaskResource, String payload)
+		throws Exception {
+
+		ImportTask importTask = importTaskResource.deleteImportTask(
+			"com.liferay.object.rest.dto.v1_0.ObjectEntry", null, null, null,
+			objectDefinition.getName(), payload);
+
+		while (true) {
```

Comment:
we can actually reuse here the waitForFinish that Carlos created. And later on I'd inlined the _executeDeleteImportTask, since we only use it once I don't think it is necessary to create this separate method 🤔